core/include/thread_flags.h

```diff
@@ -98,6 +101,20 @@ extern "C" {
  * @see xtimer_set_timeout_flag
  */
 #define THREAD_FLAG_TIMEOUT         (1u << 14)
+
+
```

Comment:
extra newline

Reply:
That was originally intentional to set it off from the others -- revisiting it, given the others don't have newlines between them, added.